plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -900,3 +900,23 @@ SecRule REQUEST_FILENAME "@endsWith /wp-json/aioseo/v1/wizard" \
     nolog,\
     ctl:ruleRemoveTargetById=930120;ARGS_NAMES,\
     ver:'wordpress-rule-exclusions-plugin/1.0.1'"
+
+
+#
+# [ Appearance ]
+#
+
+# Accessing WordPress (Appearance --> Customize) gives FP at PL 1
+# FP: "Invalid date" value in _wpCustomizeControlsL10n array in customize-controls script
+# This activates rule 953100 which looks for "Invalid date" pattern in php-errors.data
+# Tested with the latest CRS rules from default branch with tag - v4.0.0-rc1
+SecRule REQUEST_FILENAME "@beginsWith /wp-admin/customize.php" \
```

Comment:
I looked up the [CRS documentation](https://coreruleset.org/docs/concepts/false_positives_tuning/#example-5-ctlruleremovebyid) and they used `@beginsWith` for each of the example mentioned. Maybe you guys follow some other guideline. It'd be nice to have access to those.

Reply:
I followed [CRS documentation](https://coreruleset.org/docs/concepts/false_positives_tuning/#example-5-ctlruleremovebyid) and they use `@beginsWith` in most of their examples. Maybe you guys use different guidelines. It'd be nice to have access to those.

Reply:
No sure why is this in our docs but almost all exclusions rules are using `@endsWith` for very simple reason: If you use `@beginsWith`, you will prevent this exclusion rule to work for WordPress installations inside a directory (for example, URL for such installation can be `/wp/wp-admin/customize.php` and your exclusion rule will not match this).

Reply:
Anyway, you need to think everytime what is better as using `@endsWith` instead of `@beginsWith` is not some kind of universal thing to do in every situation.

Reply:
Or, maybe, better hint: It is usually better to use `@endsWith` instead of `@beginsWith` when you are matching `REQUEST_FILENAME`.